public/style.css

```diff
@@ -0,0 +1,107 @@
+body{
+  color: #02075d;
+  background-color: #afeeee;
+  font-family: arial;
+  font-size: 1.5em;
+  text-align: center;
+}
+
+h3{
+  text-align: left;
+  width: 70vw;
+}
+
+p {
+  text-align: left;
+  padding: 5px;
+  width: 70vw;
+  margin: 0;
+}
+
+.comments{
+  font-size: 0.7em;
+}
```

Comment:
I wasn't aware that you could use a stylesheet for other css classes. I will shamelessly yoink this for my other projects ;)